lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -1577,6 +1577,48 @@ Value *PatchInOutImportExport::performFsHalfInterpolation(BuilderBase &builder,
   return result;
 }
 
+// =====================================================================================================================
+// Load a specified FS parameter
+//
+// @param builder : The IR builder to to create and insert IR instruction
+// @param attr : The attribute location to access
+// @param channel : The specific attribute channel to access
+// @param interpParam : the parameter to load
```

Comment:
Nitpick: "s/the/The" to keep it consistent with the others (same for text for "bitWidth" two lines below).

Reply:
Thanks! fixed locally.